frontend/src/pages/spotnet/dont_miss/DontMiss.jsx

```diff
@@ -4,7 +4,6 @@ import { ReactComponent as Rocket } from 'assets/icons/rocket.svg';
 import { ReactComponent as Hand } from 'assets/images/hand.svg';
 import { ReactComponent as Star } from 'assets/particles/star.svg';
 import { useNavigate } from 'react-router-dom';
-import { Notifier, notify } from 'components/Notifier/Notifier';
```

Comment:
keep importing notify as it it used in the file

Reply:
@NowGoody not sure yet, but I think that build wil fail, because there are unused `Notifier` import in `DontMiss`, you should remove it and keep only notify

Reply:
@NowGoody yeah

```
src/pages/spotnet/dont_miss/DontMiss.jsx
24.20   Line 7:10:  'Notifier' is defined but never used  no-unused-vars
```

this line should be `import { notify } from 'components/Notifier/Notifier';`